apis/validator/prepare_beacon_proposer.yaml

```diff
@@ -12,7 +12,7 @@ post:
 
     Optionally the validator may also include a signed validator registration
     which will be used by the Builder API to verify a validator's latest fee
-    recipient.
+    recipient and preferred target gas limit.
```

Comment:
```suggestion
    recipient and preferred gas limit.
```

Reply:
just to make it clear which we are talking about here

Reply:
I think _target_ is most clear, when the `parent.gas_limit` is greater than `parent.gas_limit / 1024` from the validators preferred gas limit, it is not possible to build a block with the preferred limit.

Reply:
yeah I just meant that having the words `target` and `limit` together can be confusing as they are distinct concepts of managing the gas limit under 1559